### DISCUSSION TITLE
ERA5 as first Met forcing

### DISCUSSION THREAD

**Discussion by slabhan1** [NASA-LIS/LISF#1503] [Q&A]
Hello everyone!
I'm new here and I'm trying to learn using LIS NoahMP and Hymap.
- I tried to pass ERA5 met forcing without Hymap and it worked just fine with round number domain parameters (for eg, (8,25)x(72,85)). 
- However, to accommodate my Hymap domain, I cropped the domain corners by 0.025 (8.025,24.975)x(72.025,84.975). This made the LIS crash during the ERA5 reading, throwing an MPI Abort error. 
- I am now using MERRA2 as first met forcing, and overlaying ERA5 and it works. I was just wondering what was causing this and if anyone has had the same issues. 
Thank you in advance!

**Comment by UDTIAN** [comment]
Hi,
Did you solve your problem? I am currently using ERA5 meteorological data to force the LISF model. The model crashes without any bug errors after line 73, which shows "Noah-3.6 Landuse type MODI - found 20 categories." More details can be found below:
70 [INFO] Using ERA5 forcing
71 [INFO] ERA5 forcing directory: ./ERA5.FORCING
72 [INFO] Reading Noah-3.6 vegetation parameter file: ./INPUT/LS_PARAMETERS/noah36_parms/VEGPARM.TBL
73 [INFO] Noah-3.6 Landuse type MODI - found 20 categories

**Comment by slabhan1** [comment]
Hi!
I did not solve my problem but I changed the met data from ERA5 to MERRA2. My error looked different from yours as well!
________________________________
